dist/css/bootstrap.css

```diff
@@ -7948,6 +7934,20 @@ select[multiple].input-group-sm > .input-group-addon {
 .navbar-nav > li > .dropdown-menu {
   margin-top: 56px;
 }
+.navbar-nav > li > .dropdown-menu:before {
```

Comment:
Why :before?

Reply:
The `:before` is used to show the little arrow of the dropdown.
![image](https://user-images.githubusercontent.com/1814752/41838666-ee792aa8-7858-11e8-8578-733a2eddcd59.png)